core/src/replay_stage.rs

```diff
@@ -3622,6 +3638,22 @@ impl ReplayStage {
             if let Some(replay_result) = &replay_result.replay_result {
                 match replay_result {
                     Ok(replay_tx_count) => tx_count += replay_tx_count,
+                    Err(BlockstoreProcessorError::BlockComponentProcessor(
+                        BlockComponentProcessorError::AbandonedBank(update_parent),
+                    )) => {
+                        let parent_slot = match update_parent {
+                            VersionedUpdateParent::V1(x) => x.new_parent_slot,
+                        };
+
+                        Self::clear_bank(bank_forks, bank_slot);
```

Comment:
does this work without clearing the ancestor / descendents map? I guess because it's not used and reset on the next iteration?